test/acceptance/features/redis.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Redis client
+
+  Scenario: Publish and subscribe a text message
+    Given the redis endpoint
+          | addr     | localhost:6379 |
+          | db       | 0              |
+      And I subscribe to the redis topic "test-topic"
+     When I publish a message to the redis topic "test-topic" with the text
+          """
+          This is a test message
+          """
+     Then I wait up to "3" seconds for a redis message with the text
+          """
+          This is a test message
+          """
+     And I unsubscribe from the redis topic "test-topic"
+
+  Scenario: Publish and subscribe a JSON message
+    Given the redis endpoint
+          | addr     | localhost:6379 |
+          | db       | 0              |
+      And I subscribe to the redis topic "test-topic"
+     When I publish a message to the redis topic "test-topic" with the JSON properties
+          | id       | abc    |
+          | name     | Golium |
+     Then I wait up to "3" seconds for a redis message with the JSON properties
+          | id       | abc    |
+          | name     | Golium |
+     And I unsubscribe from the redis topic "test-topic"
```

Comment:
As you should unsubscribed forcely and to avoid people forgetting to add this step and get in troubles, shouldn't it be bette, to unsubscribe as a general post scenario task (toolium like)?

Reply:
Not sure how to implement that.
Meanwhile, we can maintain it as it is. There's no big impact if you don't unsubscribe.